app/scripts/services/hpa.js

```diff
@@ -185,10 +219,17 @@ angular.module("openshiftConsole")
       return hpaByResource;
     };
 
+    // currently unsupported by the webconsole
+    var isUnsupportedAPI = function(hpa) {
```

Comment:
nit: `isUnsupportedAPI` sounds like the API is no longer supported by the server, but that's not the case. I think something like `usesV2Metrics` is more clear? The resource is technically v1 API since that's what we requested.